src/utils/unbxdCallBack.js

```diff
@@ -9,7 +9,7 @@ function unbxdCallBack(unbxdSearchObj, eventName, data) {
         });
         const { onRouteChange } = this.props;
         if (typeof onRouteChange === 'function') {
-            onRouteChange(unbxdSearchObj.getStateString());
+            onRouteChange(unbxdSearchObj, unbxdSearchObj.getStateString());
```

Comment:
Why pass both, we can get second from first param, Right?

Reply:
Yes we can, but I felt it would be handy if we passed it as separate param.

Reply:
ok